e2e/__tests__/tours.test.js

```diff
@@ -0,0 +1,171 @@
+    launchDate: new Date(),
+    stops: [{}]
+  };
+
+  const location1 = {
+    name: 'Our House, In the Middle of the Street',
+    address: '97216'
+  };
+
+  const attend1 = {
+    attendance: 42
+  };
+
+  function postTour(data) {
+    return request
+      .post('/api/tours')
+      .send(data)
+      .expect(200)
+      .then(({ body }) => body);
+  }
+
+  it('posts a tour', () => {
+    return postTour(data).then(tour => {
+      expect(tour).toMatchInlineSnapshot(
+        {
+          _id: expect.any(String),
+          launchDate: expect.any(String),
+          stops: [{ _id: expect.any(String) }]
+        },
+        `
+        Object {
+          "__v": 0,
+          "_id": Any<String>,
+          "activities": Array [
+            "live music",
+            "food",
+            "camping",
+            "butts",
+          ],
+          "launchDate": Any<String>,
+          "stops": Array [
+            Object {
+              "_id": Any<String>,
+            },
+          ],
+          "title": "Don't Stop Til You Drop!",
+        }
+      `
+      );
+    });
+  });
+
+  it('gets all tours', () => {
+    return postTour(data).then(() => {
+      return request.get('/api/tours').then(({ body }) => {
+        expect(body[0]).toMatchInlineSnapshot(
+          {
+            _id: expect.any(String),
+            launchDate: expect.any(String),
+            stops: [{ _id: expect.any(String) }]
+          },
+          `
+          Object {
+            "__v": 0,
+            "_id": Any<String>,
+            "activities": Array [
+              "live music",
+              "food",
+              "camping",
+              "butts",
+            ],
+            "launchDate": Any<String>,
+            "stops": Array [
+              Object {
+                "_id": Any<String>,
+              },
+            ],
+            "title": "Don't Stop Til You Drop!",
+          }
+        `
+        );
+      });
+    });
+  });
+
+  it('adds a stop to a tour', () => {
+    return postTour(data)
+      .then(tour => {
+        return request
+          .post(`/api/tours/${tour._id}/stops`)
+          .send(location1)
+          .expect(200)
+          .then(body => {
+            return [body, location1];
+          });
+      })
+      .then(out => {
+        const stops = out[0].body[1];
+        expect(stops).toMatchInlineSnapshot(
+          {
+            _id: expect.any(String)
+          },
+          `
+          Object {
+            "_id": Any<String>,
+          }
+        `
+        );
+      });
+  });
+
+  it('removes a stop', () => {
+    return postTour(data).then(tour => {
+      return request
+        .post(`/api/tours/${tour._id}/stops`)
+        .send(location1)
+        .expect(200)
+        .then(out => {
+          const stops = out.body[1];
+          return request
+            .delete(`/api/tours/${tour._id}/stops/${stops._id}`)
+            .send(tour._id, stops._id)
+            .expect(200);
```

Comment:
you should test to make sure the stop isn't in your tour anymore (you could do this with a snapshot test)